Instructions/README.md

```diff
@@ -13,6 +13,5 @@ Ensure the front and back measurements of the left side are the same.  Then usin
 - build on a level hard surface and ensure the frame is square as you tighten the frame bolts.
 - when installing the frame braces, tighten them but don't go hulk strength and crack them.
 - I found fender washer for the y carriage bolts spread the load and help maintain consisntent contact
-- make sure you use the correct length M6 screws, most pieces use M6 x 12 but others use M6 x 14
+- make sure you use the correct length M6 screws, most pieces use M6 x 12 but others use M6 x 14, M6x12mm screws should be used for the parts that have recessed holes.  Namely: Y idler and motor holder, the Rambo mount holder and the Z Rod Holders
```

Comment:
Maybe one line per bolt size explaining where they would be used would be more clear?